src/widgets.jl

```diff
@@ -52,10 +51,14 @@ s = slider(1:10; piecewiseLabel=true)
 
 """
 function slider{T}(vals::Union{Range{T}, Vector{T}, Associative{<:Any, T}};
-                value=nothing,
-                ob=nothing,
+                value=medianelement(vals),
                 label="", kwargs...)
-    ob, value = init_wsigval(ob, value; typ=T, default=medianelement(vals))
+
+    if value isa Observable
+        ob = value
+    else
+        ob =  Observable{T}(medianelement(vals))
+    end
```

Comment:
Change to be like other components?
```
if !(value isa Observable)
     value = Observable{T}(value)
end
```
either way `ob =  Observable{T}(medianelement(vals))` should be `ob =  Observable{T}(value)` no?